r/tests/testthat/helper-expectation.R

```diff
@@ -91,7 +91,7 @@ expect_dplyr_equal <- function(expr,
 
   if (isTRUE(warning)) {
     # Special-case the simple warning:
-    warning <- "not supported in Arrow; pulling data into R"
+    warning <- "not supported (in|by) Arrow; pulling data into R"
```

Comment:
Oops, we might want to harmonise `arrow_not_supported` with it's "not supported by Arrow" with the "not supported in Arrow" phrasing used elsewhere (though I think we should do that in a separate ticket), but this regex works for now.

Reply:
Please add a TODO note (and optionally make a JIRA)